modules/rosa-policy-security-regulation-compliance.adoc

```diff
@@ -54,9 +54,9 @@ Any issues that may be discovered are prioritized based on severity. Any issues
 .Security and control certifications for {product-title}
 [cols= "3,3,3",options="header"]
 |===
-| Certification | {product-title} | {hcp-title-first}
+| Compliance | {product-title} | {hcp-title-first}
```

Comment:
there is a discrepancy in how the product titles are rendering.  On the left it is just "Red Hat OpenShift Service on AWS". On the right it has the acronym.  Do we want to make sure they are similar?

Reply:
Updated the second column as, "Red Hat OpenShift Service on AWS (ROSA)". This way the second and third column product titles are consistent.